app/i18n/locales/hr-HR.json

```diff
@@ -18,12 +18,12 @@
   "loading.screen.reconnectingToNetworkMessage": "!!!Network connection lost - reconnecting",
   "loading.screen.syncingBlocksMessage": "!!!Syncing blocks",
   "profile.languageSelect.form.languageSelectLabel": "Odaberi jezik",
-  "profile.languageSelect.form.options.chinese": "!!!Chinese",
-  "profile.languageSelect.form.options.croatian": "!!!Croatian",
-  "profile.languageSelect.form.options.english": "!!!English",
-  "profile.languageSelect.form.options.german": "!!!German",
-  "profile.languageSelect.form.options.japanese": "!!!Japanese",
-  "profile.languageSelect.form.options.korean": "!!!Korean",
+  "profile.languageSelect.form.options.chinese": "Kineski",
+  "profile.languageSelect.form.options.croatian": "Hrvatski",
+  "profile.languageSelect.form.options.english": "Engleski",
+  "profile.languageSelect.form.options.german": "Njemački",
+  "profile.languageSelect.form.options.japanese": "Japanski",
+  "profile.languageSelect.form.options.korean": "Korejski",
```

Comment:
Darko said: In this case the languages should be lowercase letter in Croatian (he thinks because it's an adjective) 😂